src/include/sof/math/log.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_TWO_LOGARITHM_H__
+#define __SOF_MATH_BASE_TWO_LOGARITHM_H__
+
+/* Include Files */
+#include <stdint.h>
+#include <stdlib.h>
```

Comment:
Maybe stdlib.h is not needed for these two small header files?

Reply:
yes, i will remove it